### ISSUE TITLE
From Error log: Bad argument @ 'getDistanceBetweenPoints3D' [Expected number, got NaN]

### ISSUE THREAD

**Issue by geramy92** [TerraTex-Community/TerraTex-Reallife-Reloaded#548]
- Database Id: 2
- Message: Bad argument @ 'getDistanceBetweenPoints3D' [Expected number, got NaN]
- Stacktrace: 
```
stack traceback:
	...erratex_reallife\SYSTEM\logging\debug_log_client.lua:3: in function <...erratex_reallife\SYSTEM\logging\debug_log_client.lua:2>
	[C]: in function 'getDistanceBetweenPoints3D'
	...tex_reallife\ENVIRO\voice_chat\voice_chat_client.lua:23: in function <...tex_reallife\ENVIRO\voice_chat\voice_chat_client.lua:3>
```
- Level: WARNING
- Source: terratex_reallife\ENVIRO\voice_chat\voice_chat_client.lua:23
- LogType: client
- Appeared at User: Samyz

**Comment by geramy92** [comment]
Causes #549

**Comment by geramy92** [comment]
Solved in #568
